ibft/controller/controller_sync.go

```diff
@@ -33,9 +34,18 @@ func (i *Controller) processSyncQueueMessages() {
 
 // ProcessSyncMessage processes sync messages
 func (i *Controller) ProcessSyncMessage(msg *network.SyncChanObj) {
+	i.logger.Info("ProcessSyncMessage: processing message")
 	var lastChangeRoundMsg *proto.SignedMessage
 	currentInstaceSeqNumber := int64(-1)
 	if i.currentInstance != nil {
+		if i.currentInstance.State() == nil {
+			i.logger.Warn("ProcessSyncMessage: i.currentInstance != nil but i.currentInstance.State() == nil")
```

Comment:
We should have a meaningful log here, e.g. `could not process message: IBFT instance doesn't have an active state`

Same for the log in line 44

Reply:
I added these both logs just to try to catch in Kibana what exactly causes the panic, so I'm going to remove both of them before merging the PR